recipes/maesters-nwp/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "maesters-nwp" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 91165b891d4d1707e08572af996080fe2376a2978f4c963203b7246bc4e8d245
+
+build:
+  skip: true  # [win]
```

Comment:
Please revert to `noarch: python` or explain why this package cannot be noarch (missing dependencies on a platform are not a reason for a package to avoid noarch).